### PR TITLE
new moose build command and update to docker build process.

### PR DESCRIPTION
**Pull request by cjus**

### CODE REVIEW

File: apps/framework-cli/src/cli.rs [hunk 1]
```diff
@@ -168,16 +184,6 @@ async fn top_command_handler(settings: Settings, commands: &Commands) {
                 controller.run_routines(run_mode);
             }
             Commands::Docker { sub_method } => match sub_method.as_str() {
```
Comment (callicles) [inline apps/framework-cli/src/cli.rs]: should we remove this all together?

**Review by callicles — COMMENTED** [review]
Quick question about former code clean up

File: apps/framework-cli/src/cli.rs (hunk 1 above)
Comment (cjus) [reply apps/framework-cli/src/cli.rs]: No we still need `moose docker build` and I imagine we'll want `moose docker push` to push to repos later. I believe @03cranec has a linear ticket for that.

File: apps/framework-cli/src/cli.rs [hunk 2]
```diff
@@ -200,51 +219,6 @@ async fn top_command_handler(settings: Settings, commands: &Commands) {
...
             }
-            Commands::Docker { sub_method } => match sub_method.as_str() {
-                "init" => {
-                    let run_mode = RunMode::Explicit {};
-                    info!("Running docker init command");
-                    let project = Project::load_from_current_dir()
-                        .expect("No project found, please run `moose init` to create a project");
-                    let project_arc = Arc::new(project);
-
-                    crate::utilities::capture::capture!(
```
Comment (callicles) [inline apps/framework-cli/src/cli.rs:211]: Looks like we are loosing this with the change - we probably want to instrument the `build` command all the same

Comment (cjus) [reply apps/framework-cli/src/cli.rs:211]: Added new capture points. I also refactored capture.rs ActivityType's to match this PRs changes.

**Review by callicles — APPROVED** [review]

**Pull request merged**